src/list-of-meals/constantOfListOfMeal.tsx

```diff
@@ -1 +1,3 @@
 export const serialNumberOfFirstMeal = 1;
+export const iconEaten =
```

Comment:
I'd remove this field from meals in the DB, because you have this link on the FE

Reply:
did